### PR TITLE
Disable gcc printf verifier for logging function

### PR DESCRIPTION
**Pull request by notspiff**
Now that the newfangled libfmt stuff is used, we get a warning from the verifier and it causes more harm than good.

### CODE REVIEW

**Comment by wsnipex** [comment]
@Montellese ping

File: xbmc/utils/log.h [hunk 1]
```diff
@@ -42,7 +42,7 @@ class CLog
   CLog();
   ~CLog(void);
   static void Close();
-  static void Log(int loglevel, PRINTF_FORMAT_STRING const char *format, ...) PARAM2_PRINTF_FORMAT;
+  static void Log(int loglevel, PRINTF_FORMAT_STRING const char *format, ...);
```
Comment (Paxxi) [inline xbmc/utils/log.h]: This should probably use a variadic template now that it passes everything on to libfmt but that's a larger change.

**Comment by Montellese** [comment]
Removing the verifier certainly makes sense. And what @Paxxi said makes sense as well but can be done in a separate PR.

**Comment by wsnipex** [comment]
jenkins build and merge

**Pull request merged**